ios/Video/RCTVideo.swift

```diff
@@ -1415,11 +1523,57 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
             }
 
             self._videoLoadStarted = false
+            self.handleMetadataUpdateForTrackChange()
             self._playerObserver.attachPlayerEventListeners()
             self.applyModifiers()
         }
     }
 
+    func handleMetadataUpdateForTrackChange() {
+        if onTextTracks != nil {
+            self.onTextTracks?(
```

Comment:
trzeba przejrzeć się temu co jest w dokumentacji https://miquido.github.io/react-native-video/component/events#ontexttracks kontra temu co jest zwracane tutaj. Bo to nie do końca wygląda na spójne.

zastanawiające jest też to że onTextTracks było jakby zaimplementowane, a tutaj wygląda jakby osobne miejsce skąd to dostarczamy. Nie możemy jakość tych miejsc pogodzić?